src/engine/MaxConstraint.cpp

```diff
@@ -531,8 +672,18 @@ String MaxConstraint::serializeToString() const
 {
     // Output format: max,f,element_1,element_2,element_3,...
     Stringf output = Stringf( "max,%u", _f );
-    for ( const auto &element : _elements )
+    for ( const auto &element : _elements ){
         output += Stringf( ",%u", element );
+    }
+    // special delimiter ",e" represents elimination flag and variables
```

Comment:
In case of no eliminated variables, maybe don't print anything? I.e., have the 'e' and the rest only appear if there are eliminated values